core-web/libs/dotcms-scss/angular/dotcms-theme/components/_galleria.scss

```diff
@@ -1 +1,32 @@
-//TODO: Implement Galleria styles.
+@use "variables" as *;
+@import "mixins";
+
+.p-galleria .p-galleria-item-nav {
+    background: rgba(255, 255, 255, 0.65);
+    color: $color-palette-gray-800;
+    width: $spacing-6;
+    height: $spacing-6;
+    transition: background-color $basic-speed, color $basic-speed, box-shadow $basic-speed;
+    border-radius: 50%;
+    margin: -$spacing-3 $spacing-1 0;
+}
+.p-galleria .p-galleria-item-nav .p-galleria-item-prev-icon,
+.p-galleria .p-galleria-item-nav .p-galleria-item-next-icon {
+    font-size: $font-size-default;
+}
+.p-galleria .p-galleria-item-nav .p-icon-wrapper .p-icon {
+    width: $font-size-default;
+    height: $font-size-default;
+}
+.p-galleria .p-galleria-item-nav:not(.p-disabled):hover {
+    background: rgba(255, 255, 255, 0.85);
+}
+
+.p-galleria .p-galleria-item-nav.p-disabled {
+    background: rgba(255, 255, 255, 0.65);
+    color: $color-palette-gray-500;
+}
+
+.p-galleria .p-galleria-item-nav:focus {
+    @include field-focus;
+}
```

Comment:
I think we should make use of `scss` nested syntax since is easier to read and easier to maintain I made this example on the fly of how it could be re-written. Please make sure it works. 

```scss
@use "variables" as *;
@import "mixins";

.p-galleria {

  .p-galleria-item-nav {
    background: rgba(255, 255, 255, 0.65);
    color: $color-palette-gray-800;
    width: $spacing-6;
    height: $spacing-6;
    transition: background-color $basic-speed, color $basic-speed, box-shadow $basic-speed;
    border-radius: 50%;
    margin: $spacing-3 $spacing-1 0;

    .p-icon-wrapper .p-icon {
      width: $font-size-default;
      height: $font-size-default;
    }

    &:not(.p-disabled):hover {
      background: rgba(255, 255, 255, 0.85);
    }

    &.p-disabled {
      background: rgba(255, 255, 255, 0.65);
      color: $color-palette-gray-500;
    }

    &:focus {
      @include field-focus;
    }

    & .p-galleria-item-prev-icon,
    & .p-galleria-item-next-icon {
      font-size: $font-size-default;
    }
  }
}
```

If you have any question don't hesitate to ping me

Reply:
done